iconrpcserver/dispatcher/default/websocket.py

```diff
@@ -104,56 +105,50 @@ class WSDispatcher:
 
     @staticmethod
     async def publish_heartbeat(ws):
-        exception = None
-        while ws.open:
+        call_method = "node_ws_PublishHeartbeat"
+        while True:
             try:
-                request = Request("node_ws_PublishHeartbeat")
-                Logger.debug(f"node_ws_PublishHeartbeat: {request}")
+                request = Request(call_method)
+                Logger.debug(f"{call_method}: {request}")
                 await ws.send(json.dumps(request))
                 heartbeat_time = StubCollection().conf[ConfigKey.WS_HEARTBEAT_TIME]
                 await asyncio.sleep(heartbeat_time)
+            except exceptions.ConnectionClosed:
+                Logger.debug("Connection Closed by child.")  # TODO: Useful message needed.
```

Comment:
How about `break` at finally block? 
I tested like below: 

[nav] In [5]: while True:
         ...:     try:
         ...:         print(1)
         ...:         raise RuntimeError()
         ...:     except:
         ...:         print("E!")
         ...:     finally:
         ...:         print("fin!")
         ...:         break
1
E!
fin!

Reply:
I changed like this.
It will catch all exceptions in loop. and loop breaks when any of exception is raised.

```
try:
    while loop
except:
    handles
```